src/components/FileDetail/VersionItem.js

```diff
@@ -0,0 +1,153 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classes from 'classnames';
+import {Mutation} from 'react-apollo';
+import {FILE_DOWNLOAD_URL} from '../../state/mutations';
+import {KF_STUDY_API} from '../../common/globals';
+import {Avatar, Icon} from 'kf-uikit';
+import TimeAgo from 'react-timeago';
+import CopyButton from '../CopyButton/CopyButton';
+import {fileTypeDetail, formatFileSize} from '../../common/fileUtils';
+/**
+ * Displays single version item from the list
+ */
+const VersionItem = ({
+  className,
+  studyId,
+  fileId,
+  fileType,
+  fileName,
+  versionNode,
+  index,
+}) => {
+  const ext = fileName.substr(fileName.lastIndexOf('.'));
+  let versionItemClass = classes('FileVersionList--Element', className);
+  return (
+    <Mutation mutation={FILE_DOWNLOAD_URL}>
+      {downloadFile => (
+        <li key={versionNode.kfId} className={versionItemClass}>
+          <svg
```

Comment:
let's move the svgs to their own file https://facebook.github.io/create-react-app/docs/adding-images-fonts-and-files#adding-svgs

Reply:
Will be addressed in future pr, issue created #243